deps/curl-8.6.0/winbuild/MakefileBuild.vc

```diff
@@ -48,11 +48,11 @@ CC = cl.exe
 !IF "$(VC)"=="6"
 CC_NODEBUG  = $(CC) /O2 /DNDEBUG
 CC_DEBUG    = $(CC) /Od /Gm /Zi /D_DEBUG /GZ
-CFLAGS      = /I. /I../lib /I../include /nologo /W4 /GX /DWIN32 /YX /FD /c /DBUILDING_LIBCURL /ZH:SHA_256 /guard:cf /Qspectre /sdl
+CFLAGS      = /I. /I../lib /I../include /nologo /W4 /GX /DWIN32 /YX /FD /c /DBUILDING_LIBCURL /Z7 /ZH:SHA_256 /guard:cf /Qspectre /sdl

```

Comment:
don't we need to add /W3 as for azuresdk and zlib?

Reply:
It's using /W4 already.